src/mlpack/methods/ann/activation_functions/elliot_function.hpp

```diff
@@ -0,0 +1,95 @@
+/**
+ * @file elliot_function.hpp
+ * @author Bisakh Mondal
+ *
+ * Definition and implementation of the Elliot Activation function as described 
+ * by D.L. Elliott
+ *
+ * For more information, see the following paper.
+ *
+ * @code
+ * @techreport{elliott1993better,
```

Comment:
Hi @bisakhmondal, Shouldn't this be `Elliot1993`. However I might be missing something.

Reply:
Okay changing it... I just copied the BibTex of the paper.